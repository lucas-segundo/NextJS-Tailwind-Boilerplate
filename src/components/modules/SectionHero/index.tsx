import Box from 'components/elements/Box'

const SectionHero = () => (
  <section>
    <h1>SectionHero</h1>
    <Box />
  </section>
)

export default SectionHero
