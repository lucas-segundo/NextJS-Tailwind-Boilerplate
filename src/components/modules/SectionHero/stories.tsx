import { Story, Meta } from '@storybook/react'
import SectionHero from '.'

export default {
  title: 'SectionHero',
  component: SectionHero
} as Meta

export const Default: Story = () => <SectionHero />
