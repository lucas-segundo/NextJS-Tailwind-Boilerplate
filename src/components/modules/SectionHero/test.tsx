import { render, screen } from '@testing-library/react'

import SectionHero from '.'

describe('<SectionHero />', () => {
  it('should render the heading', () => {
    const { container } = render(<SectionHero />)

    expect(
      screen.getByRole('heading', { name: /SectionHero/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
