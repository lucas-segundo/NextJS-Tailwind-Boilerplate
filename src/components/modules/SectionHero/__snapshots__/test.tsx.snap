// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionHero /> should render the heading 1`] = `
<section>
  <h1>
    SectionHero
  </h1>
  <section>
    <h1>
      Box
    </h1>
  </section>
</section>
`;
